requirements-rtd.txt

```diff
@@ -1,7 +1,9 @@
-myst_parser==0.18.0
 Jinja2<3.1
 Sphinx==5.1.1
 sphinx_rtd_theme==1.0.0
 readthedocs-sphinx-search==0.1.2
 sphinx-toggleprompt==0.2.0
 sphinx-copybutton==0.5.0
+spikeinterface>=0.95.1  # Necessary to display tools.spikeinterface
+spikeextractors>=0.9.10 # Necessary to display tools.spikeinterface
+roiextractors>=0.5.0  # Necessary to displays tools.roiextractors
```

Comment:
Can't we just add `-full` to the extras of the `.readthedocs.yaml`?

Reply:
I think that can be done, do you think is preferable?

Reply:
Yeh, keeps the requirements files from non overlapping (esp. w.r.t. the new recursive setup logic)

Reply:
I actually thought that you would not like to have as many unnecessary dependencies on the documentation environment. I think it makes it more brittle. As in, if any of all of the dependencies in full breaks or fails to install then the documentation will fail to build. This would not be a problem if we had a dev branch separated from a stable branch.

What do you think?